app/src/forms/RegistrationForm.py

```diff
@@ -0,0 +1,9 @@
+from wtforms import Form, StringField, PasswordField, validators
+from app.src.validators import email_validator, password_validator, date_validator
+from wtforms.fields.html5 import DateField
+
+class RegistrationForm(Form):
```

Comment:
whats Form? if it is variable then it is unused?

Reply:
it's python syntax for inheritance, Form is a class, and RegistrationForm inheritance from it.